app/src/main/java/com/besome/sketch/design/DesignActivity.java

```diff
@@ -1284,10 +1281,8 @@ private void doInBackground() {
                     dialog.show();
                 });
             } catch (zy zy) {
-                isBuildFinished = true;
```

Comment:
This should be included I think

Reply:
@ilyassesalama

Reply:
I believe those 2 lines weren't removed with the commit I just committed recently (?)